Dapper/DefaultTypeMap.cs

```diff
@@ -29,22 +29,22 @@ public DefaultTypeMap(Type type)
 
         internal static MethodInfo GetPropertySetter(PropertyInfo propertyInfo, Type type)

```

Comment:
Not sure how it interacts with the direct emits, but feels like this is more correct as `MethodInfo?`